chromium_src/extensions/browser/extension_action_manager.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/common/extensions/constants.h"
```

Comment:
`//extensions/browser` cannot have a dependency on `//brave/common/*`
I think both `brave/common/extensions/constants.h` and `brave/common/extensions/extension_constants.h` need to go in `extensions/common/constants.h` instead

Reply:
Addressed by 9688438ad52c380d21f0f04f9819ce8e7c7112dc